src/service/packet_router.proto

```diff
@@ -6,6 +6,11 @@ import "region.proto";
 import "data_rate.proto";
 
 message packet_router_packet_report_v1 {
+  enum packet_type {
+    join = 0;
+    uplink = 1;
+  }
+
   uint64 gateway_timestamp_ms = 1;
```

Comment:
rename to `tmst` to reflect that it means in packet forwarder land? Name changes are ok for already running systems